app/controllers/answers_controller.rb

```diff
@@ -1,2 +1,28 @@
 class AnswersController < ApplicationController
-end
+    def index
```

Comment:
Regarding format, let's add empty lines between methods for legibility.

And consider discussing with the group indent settings: it's convention to use 2 spaces in Rails, which you can set in your code editor (Settings -> Tab Size)